### PR TITLE
Feature/63 appearing text extension

### LINKED ISSUE #63: Appearing text extension
## Description

When displaying appearing text, it should sometimes appear instantly. For this, the APPEAR_INSTANTLY action should be added (or something similair), which makes the next spoken line appear instantly.

Also, sometimes the textbox needs to be hidden. For this, the HIDE_TEXTBOX action should be added, which hides the textbox until a new sentence is spoken, after which it is shown again.

## Acceptance Criteria

This story will be finished when the aforementioned actions have been added and are testable in a unity scene

### PR DESCRIPTION
**Pull request by PhoebeMitchell**
## Summary

Implements HIDE_TEXTBOX and APPEAR_INSTANTLY actions, which hide the dialogue textbox and make text appear in the dialogue textbox instantly respectively.

## Before/after screenshots and/or animated gif

instantlyappearingtextdemo

## Testing instructions

Open "Inky-Test Scene", assign "InstantlyAppearingTextDemo" as the dialogue controller's narrative scripts, and enter play mode.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
#63

### CODE REVIEW

**Review by notexplosive — DISMISSED** [review]
goob job

File: Assets/Scripts/UI/AppearingDialogueController.cs [hunk 1]
```diff
@@ -537,8 +548,26 @@ private void ChangeNameBGColor(Color newColor)
...
+    /// </summary>
+    public void HideTextbox()
+    {
+        _textBoxGameObject.SetActive(false);
+    }
 
-
+    /// <summary>
+    /// Shows the dialogue textbox.
+    /// Should be subscribed to DialogueController's OnNewSpokenLine event.
```
Comment (JuiceGrape) [inline Assets/Scripts/UI/AppearingDialogueController.cs]: Seems a bit over-complicated. Isn't there a function in this controller that is already attached to that event?

**Review by JuiceGrape — DISMISSED** [review]
Looks good

File: Assets/Scripts/UI/AppearingDialogueController.cs (hunk 1 above)
Comment (PhoebeMitchell) [reply Assets/Scripts/UI/AppearingDialogueController.cs]: Good point. Moved the contents of this method to StartDialog

**Review by JuiceGrape — APPROVED** [review]
Please don't forget to add your changes to the wiki :)

**Review by notexplosive — APPROVED** [review]

**Pull request merged**